scripts/amd/gemm/tune_gemm.py

```diff
@@ -165,16 +181,18 @@ def gen_kernel_and_configStr_from_config(M, N, K, config, dtype_a, dtype_b, dtyp
     if dtype_c:
         torch_dtype_c = tl_to_torch_types[name_to_tl_types[dtype_c]]
     configStr = f"M{M}_N{N}_K{K}_BM{block_m}_BN{block_n}_BK{block_k}_GM{group_m}_SK{split_k}_nW{num_warps}_nS{num_stages}_EU{waves_per_eu}_kP{kpack}_mfma{mfmaInstrSize}"
-
+    if bias_size > 0:
+        configStr += "_bias"
```

Comment:
I think we don't need `_bias` to appear in the configStr. We only need tuning parameters. 
So actually we don't even need gemm sizes.

Reply:
Yes, we are planning another change to tune multiple shapes together to reduce compile time. Maybe we can remove the shapes and bias info then.

Reply:
How does tuning multiple shapes together reduce compile time? Are you saying reusing the config for different shapes?

Reply:
Yes, that is what I mean. We do not have to hard-coded M, N, K, so the same kernel can run different input shapes.